addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1189,12 +1185,18 @@ registerModel({
             if (!this.currentRtcSession.isMute) {
                 this.messaging.soundEffects.pushToTalkOff.play();
             }
-            this._pushToTalkTimeoutId = browser.setTimeout(
-                () => {
-                    this._setSoundBroadcast(false);
-                },
-                this.messaging.userSetting.voiceActiveDuration || 0,
-            );
+            this.update({
+                pushToTalkTimeoutId: browser.setTimeout(
+                    this._onPushToTalkTimeout,
+                    this.messaging.userSetting.voiceActiveDuration || 0,
```

Comment:
To check with TSO, why `|| 0`? 0 is the default value, does it ever become false, undefined or something else?

Reply:
Not sure, it's received by server in `UserSetting/convertData` and also changed in UI with a `on-change` on range `input`, value being `parseInt()`.

Reply:
done